deploy/docker-compose/README.md

```diff
@@ -0,0 +1,390 @@
+# MONAI Deploy Lite
+
+MONAI Deploy Lite provides a simple & lightweight solution for running MONAI Applications (MAPs) on a minimal MONAI Deploy platform.
+
+It is meant to bridge the big gap between the local development of MAPs in a laptop using the MONAI Deploy App SDK and a production-ready deployment. 
+
+Users of MONAI Deploy Lite will be able to run their MAPs, connected to a test PACS, or their own test/research PACS, for further validation, confidently taking steps towards production.   
+
+Reusing the same essential core services for DICOM I/O and AI workflow orchestration provides the same functionality and consistent experience independently of where and how the applications are run, with minimal changes for the end user.
+
+
+## Prerequisites
+
+### Ubuntu 20.04 or later
+
+- [Docker Engine](https://docs.docker.com/engine/install/) v20.10.18+
+- [Docker Compose Plug-in](https://docs.docker.com/compose/install/) v2.10.2+
+- [NVIDIA Container Toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html)
+- MONAI Deploy MAPs built using the [MONAI Deploy App SDK](https://github.com/Project-MONAI/monai-deploy-app-sdk)
+- (Optional) [Informatics Gateway CLI](https://github.com/Project-MONAI/monai-deploy-informatics-gateway/releases)
+
+### Windows Subsystem for Linux v2
+
+- [WSLv2](https://docs.microsoft.com/en-us/windows/wsl/install)
+- [Docker Desktop](https://docs.docker.com/desktop/install/windows-install/)
+  - With `Use the WSL 2 based engine` option enabled
+- [Ubuntu 22.04](https://www.microsoft.com/store/productId/9PN20MSR04DW) for WSL 2
+- [NVIDIA Windows Driver](https://docs.nvidia.com/cuda/wsl-user-guide/index.html) for WSL 2
+- (Optional) [Informatics Gateway CLI](https://github.com/Project-MONAI/monai-deploy-informatics-gateway/releases)
+
+
+**IMPORTANT: Please ensure all prerequisites are up-to-date.**
+
+*Note: see [tips](#tips--hints) section for additional instructions.*
+
+## Installation
+
+To use MONAI Deploy Lite, install all prerequisites & download this entire directory from [GitHub](https://github.com/Project-MONAI/monai-deploy/tree/main/docker-compose/deploy/docker-compos). 
+
+## Start the Services
+
+The [docker compose file](docker-compose.yml) spins up the following services. Services are accessible at the IP addresses and ports listed below and may be modified in the [.env](.env) configuration file.
+
+Execute the following command to bring up all services.
+
+```bash
+docker compose up
+docker compose up -d # or run detached
```

Comment:
Please add a sentence before the commands that explains how long this command typically takes so users know what to expect.

Reply:
updated section